Persistence/Database/DatabaseContext.cs

```diff
@@ -0,0 +1,176 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+using shipping_service.Persistence.Entities;
+
+namespace shipping_service.Persistence.Database
+{
+    public class DatabaseContext : DbContext
+    {
+        public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options)
```

Comment:
is empty constructor needed?

Reply:
This is needed to pass data to base class.